addons/udes_priorities/models/priority.py

```diff
@@ -84,45 +107,40 @@ def _has_duplicate_names_for_same_pick_type(self):
     @api.one
     def _check_for_duplicate_names_for_same_pick_type(self):
         if self._has_duplicate_names_for_same_pick_type() > 0:
-            raise UserError("Name must be unique for the picking types it is shown on")
+            raise UserError("Name must be unique for the transfer types it is shown on")
 
     @api.multi
-    def _number_of_oustanding_pickings(self):
-        Picking = self.env["stock.picking"]
-        self.ensure_one()
-        return Picking.search_count(
-            [("priority", "=", self.reference), ("state", "not in", ("cancel", "done")),]
-        )
-
-    @api.multi
-    def _check_for_oustanding_pickings(self):
+    def _check_for_outstanding_pickings(self):
 
         install_mode = self.env.context.get("install_mode", False)
 
         if install_mode:
             return
 
-        priorities_with_outstanding_picks = self.browse()
-        for priority in self:
-            if priority._number_of_oustanding_pickings() > 0:
-                priorities_with_outstanding_picks |= priority
+        privileged_user = (
+            self.env.uid == SUPERUSER_ID
+            or self.env.user.has_group("udes_security.group_debug_user")
+            or self.env.user.has_group("udes_security.group_trusted_user")
```

Comment:
Now that we're using some user groups from the udes_security module, we'll need to add the udes_security module as a dependency in the manifest.

Reply:
Don't mind adding it explicitly in this module, but we have a dependency on `udes_security` in `udes_stock`, which is a dependency for this one. :)

Reply:
I like explicit dependencies when ids are used.